xds/internal/xdsclient/xdsresource/type_cds.go

```diff
@@ -35,6 +37,18 @@ const (
 	ClusterTypeAggregate
 )
 
+// ClusterLRSServerConfigType is the type of LRS server config.
+type ClusterLRSServerConfigType int
```

Comment:
Yes, this is a forward looking change.

The spec actually says this field should be the authority service config itself. But that adds complexity, because this function doesn't have access to the xdsclient or the bootstrap file.

I think making it as close to the proto message as possible is future proof.